inbm/dispatcher-agent/tests/unit/fota/test_guid.py

```diff
@@ -1,21 +1,41 @@
 from unittest import TestCase
+from mock import patch
 
 from unit.common.mock_resources import *
-from dispatcher.fota.guid import _parse_guid
+from dispatcher.fota.guid import _parse_guid, extract_guid
+from dispatcher.fota.fota_error import FotaError
 
 
 class TestGuid(TestCase):
-    def test_parse_guid(self) -> None:
+
+    @patch('inbm_common_lib.shell_runner.PseudoShellRunner.run',
```

Comment:
if this is calling extract_guid in disptacher/fota/guid.py, it should be patching dispatcher.fota.guid.PseudoShellRunner -- see https://docs.python.org/3/library/unittest.mock.html#where-to-patch (which says: The basic principle is that you should patch where an object is looked up, which is not necessarily the same place as where it is defined.)  Same for the patches below.